### ISSUE TITLE
Accessibility - Contrast issues and other accessibility concerns should be addressed.

### ISSUE THREAD

**Issue by samkhodak** [WonderTix/WonderTix#244]
Contrast issues are most evident when looking at the ticket purchasing page for any given event. It is very hard to read the button text, especially when they are disabled. There are other contrast errors with buttons across the program.

Every page and form should be looked at with an accessibility checker to at least catch obvious errors. Some forms in the project have improper labels and other bad practice errors.

**Comment by ben16williams** [comment]
I think we're taking these as them come, so in the interest of cleaning up the backlog I'm going to close this
